From ea7d7360edc541d2a07ddfeb528065808f3d1d5d Mon Sep 17 00:00:00 2001
From: Aleksey Lim <alsroot@member.fsf.org>
Date: Tue, 20 Oct 2009 13:14:18 +0000
Subject: Pippy module not in correct path #1058

---
 pippy_app.py |   12 ++++++------
 1 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/pippy_app.py b/pippy_app.py
index abaf397..bd574b4 100644
--- a/pippy_app.py
+++ b/pippy_app.py
@@ -305,12 +305,12 @@ class PippyActivity(ViewSourceActivity):
         copy2('%s/activity.py' % get_bundle_path(),
               '%s/tmp/activity.py' % self.get_activity_root())
 
-        self._pid = self._vte.fork_command \
-                    (command="/bin/sh",
-                     argv=["/bin/sh", "-c",
-                           "python %s; sleep 1" % pippy_app_name],
-                     envv=["PYTHONPATH=%s/library:%s" % (get_bundle_path(), os.getenv("PYTHONPATH", ""))],
-                     directory=get_bundle_path())
+        self._pid = self._vte.fork_command(
+                command='env',
+                argv=['env', 'PYTHONPATH=%s/library:%s' % \
+                        (get_bundle_path(), os.getenv('PYTHONPATH', '')),
+                        'python', pippy_app_name],
+                directory=get_bundle_path())
 
     def stopbutton_cb(self, button):
         try:
-- 
1.6.5.rc2

