--- Pippy-25/pippy_app.py.orig	2008-06-10 15:43:33.000000000 +0000
+++ Pippy-25/pippy_app.py	2008-11-22 04:23:29.000000000 +0000
@@ -83,7 +83,7 @@
         treecolumn = gtk.TreeViewColumn(_("Examples"), cellrenderer, text=1)
         treeview.get_selection().connect("changed", self.selection_cb)
         treeview.append_column(treecolumn)
-        treeview.set_size_request(220, 900)
+        treeview.set_size_request(270, 900)
 
         # Create scrollbars around the view.
         scrolled = gtk.ScrolledWindow()
@@ -159,19 +159,19 @@
         gobutton = gtk.Button(label=_("_Run!"))
         gobutton.set_image(goicon)
         gobutton.connect('clicked', self.gobutton_cb)
-        gobutton.set_size_request(650, 2)
+        gobutton.set_size_request(550, -1)
         buttonhbox.pack_start(gobutton)
 
         # The "stop" button
         stopbutton = gtk.Button(stock=gtk.STOCK_STOP)
         stopbutton.connect('clicked', self.stopbutton_cb)
-        stopbutton.set_size_request(200, 2)
+        stopbutton.set_size_request(200, -1)
         buttonhbox.pack_start(stopbutton)
 
         # The "clear" button
         clearbutton = gtk.Button(stock=gtk.STOCK_CLEAR)
         clearbutton.connect('clicked', self.clearbutton_cb)
-        clearbutton.set_size_request(150, 2)
+        clearbutton.set_size_request(150, -1)
         buttonhbox.pack_end(clearbutton)
 
         vbox.pack_start(buttonhbox)
